storage/bucket.go

```diff
@@ -1079,6 +1103,10 @@ type BucketAttrsToUpdate struct {
 	// more information.
 	RPO RPO
 
+	// If set, updates the autoclass configuration of the bucket.
+	// See <TBD> for more information.
```

Comment:
Remember to fill this in before we merge.

Reply:
done ;)